File: runtime/examples/Java_Code/OrderedArray/OrderedApp.java
/* Ordered Array test
*  adapted from (Estruturas de Dados & Algoritmos em Java 2ed)
*/

public class OrderedApp {
    public static void main(String[] args) {
        int maxSize = 100;
        int[] vet = new int[maxSize];
        OrdArray arr;
        arr = new OrdArray();
        arr.insert(77, vet);
        arr.insert(99, vet);
        int searchKey = 55;
        int p = arr.find(searchKey, vet);
        System.out.printf("%d\n", p);
        arr.display(vet);
    }
}

File: runtime/output/objcode/OrderedApp.c
#include <stdlib.h>
#include <stdio.h>
#include"OrdArray.c"

struct OrderedApp
{
};
void main();
struct OrderedApp *OrderedApp()
{
    struct OrderedApp *_this = malloc(sizeof(struct OrderedApp));
    return _this;
}
void main()
{
    int maxSize = 100;
    struct OrdArray *arr;
    arr = OrdArray(maxSize);
    arr->insert(77, arr);
    arr->insert(99, arr);
    int searchKey = 55;
    int p = arr->find(searchKey, arr);
}